src/NavList/__snapshots__/NavList.test.tsx.snap

```diff
@@ -23,6 +23,54 @@ exports[`NavList renders a simple list 1`] = `
   flex-grow: 1;
 }
 
+.c3 {
```

Comment:
@colebemis After exporting PageHeader, I updated the snapshots and for some reason, NavList snapshot has been affected by this, just the order of the classes though. I have encountered this before and no idea why it happened. I remember chatting with @siddharthkp about it as well. Do you know what might be going on here? As far as I see, it would be harmless to update the snapshot as there is no _actual_ change but would be great to understand why this is happening.

Reply:
Hmm, that's strange. I'm not sure why that happens. Feel free to merge though